backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/map/BpkIconMapMarker.kt

```diff
@@ -55,8 +58,8 @@ fun BpkIconMapMarker(
     state: MarkerState = rememberMarkerState(),
     tag: Any? = null,
     visible: Boolean = true,
-    zIndex: Float = 0.0f,
-    onClick: (Marker) -> Unit = {},
+    zIndex: Float? = null,
```

Comment:
made this nullable, so it can either be configured if needed or takes default behaviour for focused vs unfocused markers. we still want consumers to be able to change this as there might be many overlapping markers & different types of markers on the screen and we don't control the importance/priority between them, however a sensible default behaviour is needed